ci/terraform/interventions-api-stub/interventions-api-stub-gateway.tf

```diff
@@ -1,17 +1,13 @@
 data "aws_vpc" "auth_shared_vpc" {
   filter {
     name   = "tag:Name"
-    values = ["${var.environment}-shared-vpc"]
+    values = [var.environment == "authdev2" ? "dev-shared-vpc" : "${var.environment}-shared-vpc"]
   }
 }
 
 data "aws_vpc_endpoint" "auth_api_vpc_endpoint" {
   vpc_id       = data.aws_vpc.auth_shared_vpc.id
   service_name = "com.amazonaws.eu-west-2.execute-api"
-  tags = {
```

Comment:
merge issue with tags?

Reply:
this should be retained.

I wonder if there should be a separate variable for all these 'shared vpc environments'.

```hcl
variable "vpc_environment" {
  description = "The name of the environment this environment is sharing the VPC of. Likely this should remain null"
  type = string
  default = null
}

locals {
  vpc_environment = var.vpc_environment == null ? var.environment : var.vpc_environment
}
```

and then use `local.vpc_environment` everywhere we're doing VPC stuff.

This also applies for the common state buckets.